vllm/utils.py

```diff
@@ -10,7 +10,7 @@
 from functools import lru_cache, partial
 from platform import uname
 from typing import (Any, Awaitable, Callable, Generic, Hashable, List,
-                    Optional, Tuple, TypeVar, Union)
+                    Optional, Tuple, TypeVar, Union, Dict)
```

Comment:
nit: let's sort this alphabetically?
```suggestion
from typing import (Any, Awaitable, Callable, Dict, Generic, Hashable, List,
                     Optional, Tuple, TypeVar, Union)
```

Reply:
you are right, I modified it